### ISSUE TITLE
Recurrent Neural Network Regularization

### ISSUE THREAD

**Issue by mingqook** [mingqook/Reading_Paper#47]
## Abstract (초록) 📝
We present a simple regularization technique for Recurrent Neural Networks (RNNs) with Long Short-Term Memory (LSTM) units. Dropout, the most successful technique for regularizing neural networks, does not work well with RNNs and LSTMs. In this paper, we show how to correctly apply dropout to LSTMs, and show that it substantially reduces overfitting on a variety of tasks. These tasks include language modeling, speech recognition, image caption generation, and machine translation.

## 논문(Paper)에서 다루는 핵심 개념 (Idea) 💡
* Dropout for RNN, Regularization

## 논문(Paper) 정리 
* https://velog.io/@mingqook/Recurrent-Neural-Network-Regularization

## 논문(Paper) 원본
* https://arxiv.org/abs/1409.2329
